src/entity/Address.ts

```diff
@@ -0,0 +1,32 @@
+import { Entity, PrimaryGeneratedColumn, Column, ManyToOne } from 'typeorm';
+import { User } from './User';
+
+@Entity()
+export class Address {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  postalCode: number;
+
+  @Column()
+  street: string;
+
+  @Column()
+  streetNumber: number;
+
+  @Column()
+  complement?: string;
```

Comment:
como esse campo é opcional, vc tem que dizer pro banco isso adicionando a flag nullable na coluna
```suggestion
  @Column({ nullable: true })
  complement?: string;
```